constants/ClassGraph.js

```diff
@@ -0,0 +1,47 @@
+export function ClassGraph () {
```

Comment:
I still don't understand why the constants are functions... They should be _const_. , that's the name of their folder. I'll take care of it in the refactor.